corehq/apps/users/migrations/0059_add_web_apps_groups_permission_flag.py

```diff
@@ -0,0 +1,22 @@
+from django.db import migrations
+
+from corehq.apps.cloudcare.models import ApplicationAccess
+from corehq.toggles import NAMESPACE_DOMAIN, WEB_APPS_PERMISSIONS_VIA_GROUPS
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+@skip_on_fresh_install
+def enable_flag(apps, schema_editor):
+    for access in ApplicationAccess.objects.filter(restrict=True):
+        WEB_APPS_PERMISSIONS_VIA_GROUPS.set(access.domain, enabled=True, namespace=NAMESPACE_DOMAIN)
```

Comment:
are you anticipating this being fast enough to run during the deploy?

Reply:
yeah, with only a couple hundred `ApplicationAccess` objects on prod this should be fast